<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2b2b2b;
}

QLineEdit {
    background-color: #1e1e1e;
    color: #ffffff;
    border: 2px solid #404040;
    border-radius: 8px;
    padding: 10px;
    font-size: 24px;
    font-weight: bold;
}

QPushButton {
    background-color: #404040;
    color: #ffffff;
    border: 2px solid #555555;
    border-radius: 8px;
    font-size: 18px;
    font-weight: bold;
    min-height: 50px;
}

QPushButton:hover {
    background-color: #555555;
    border-color: #666666;
}

QPushButton:pressed {
    background-color: #333333;
}

QPushButton#numberButton {
    background-color: #2d2d2d;
}

QPushButton#operatorButton {
    background-color: #ff9500;
    color: #000000;
}

QPushButton#operatorButton:hover {
    background-color: #ffad33;
}

QPushButton#functionButton {
    background-color: #666666;
}

QPushButton#functionButton:hover {
    background-color: #777777;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="display">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight</set>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="buttonLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>C</string>
        </property>
        <property name="objectName">
         <string>functionButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="backspaceButton">
        <property name="text">
         <string>⌫</string>
        </property>
        <property name="objectName">
         <string>functionButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="leftParenButton">
        <property name="text">
         <string>(</string>
        </property>
        <property name="objectName">
         <string>operatorButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="rightParenButton">
        <property name="text">
         <string>)</string>
        </property>
        <property name="objectName">
         <string>operatorButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button7">
        <property name="text">
         <string>7</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button8">
        <property name="text">
         <string>8</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="button9">
        <property name="text">
         <string>9</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="divideButton">
        <property name="text">
         <string>÷</string>
        </property>
        <property name="objectName">
         <string>operatorButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="button4">
        <property name="text">
         <string>4</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="button5">
        <property name="text">
         <string>5</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="button6">
        <property name="text">
         <string>6</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="multiplyButton">
        <property name="text">
         <string>×</string>
        </property>
        <property name="objectName">
         <string>operatorButton</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="button1">
        <property name="text">
         <string>1</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="button2">
        <property name="text">
         <string>2</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="button3">
        <property name="text">
         <string>3</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="subtractButton">
        <property name="text">
         <string>-</string>
        </property>
        <property name="objectName">
         <string>operatorButton</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="button0">
        <property name="text">
         <string>0</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="decimalButton">
        <property name="text">
         <string>.</string>
        </property>
        <property name="objectName">
         <string>numberButton</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="equalsButton">
        <property name="text">
         <string>=</string>
        </property>
        <property name="objectName">
         <string>operatorButton</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>+</string>
        </property>
        <property name="objectName">
         <string>operatorButton</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="squareButton">
        <property name="text">
         <string>x²</string>
        </property>
        <property name="objectName">
         <string>functionButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
